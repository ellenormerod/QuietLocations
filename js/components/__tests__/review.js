import React from 'react';
import Review from '../review';
import renderer from 'react-test-renderer';

const fakeProps = {
  review: {
    comments: 'Super great bar, would come back due to noise level',
    noise: 3
  },
}


it('renders Review using Snapshot', () => {
  expect(renderer.create(
    <Review {...fakeProps} />
  )).toMatchSnapshot()
})
