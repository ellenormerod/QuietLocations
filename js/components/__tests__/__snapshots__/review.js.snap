// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Review using Snapshot 1`] = `
<View>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "borderColor": "#e1e8ee",
          "borderWidth": 1,
          "margin": 15,
          "marginBottom": 0,
          "padding": 15,
          "shadowColor": "rgba(0,0,0, .2)",
          "shadowOffset": Object {
            "height": 0,
            "width": 0,
          },
          "shadowOpacity": 1,
          "shadowRadius": 1,
        },
        undefined,
        undefined,
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          undefined,
          undefined,
        ]
      }
    >
      <View
        pointerEvents="auto"
        style={
          Object {
            "paddingVertical": 10,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "column",
              "justifyContent": "center",
              "paddingBottom": 5,
            }
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flexDirection": "row",
                "justifyContent": "center",
                "paddingBottom": 5,
              }
            }
          >
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Object {
                    "color": "#34495e",
                    "fontFamily": "Trebuchet MS",
                    "fontSize": 15,
                    "textAlign": "center",
                  },
                ]
              }
            >
              Rating: 
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Array [
                    Object {
                      "fontFamily": "Trebuchet MS",
                      "fontSize": 30,
                      "textAlign": "center",
                    },
                    Object {
                      "color": "#f1c40f",
                    },
                  ],
                ]
              }
            >
              3
            </Text>
            <Text
              accessible={true}
              allowFontScaling={true}
              ellipsizeMode="tail"
              style={
                Array [
                  Object {},
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  undefined,
                  Object {
                    "color": "#34495e",
                    "fontFamily": "Trebuchet MS",
                    "fontSize": 18,
                    "textAlign": "center",
                  },
                ]
              }
            >
              /
              5
            </Text>
          </View>
          <View />
        </View>
        <View
          onMoveShouldSetResponder={[Function]}
          onMoveShouldSetResponderCapture={[Function]}
          onResponderEnd={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderReject={[Function]}
          onResponderRelease={[Function]}
          onResponderStart={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          onStartShouldSetResponderCapture={[Function]}
          style={
            Object {
              "flexDirection": "row",
            }
          }
        >
          <View
            style={
              Object {
                "flexDirection": "row",
                "left": 0,
                "position": "absolute",
                "top": 0,
              }
            }
          >
            <View
              collapsable={undefined}
              style={
                Object {
                  "backgroundColor": "#f1c40f",
                  "height": 20,
                  "width": 60,
                }
              }
            />
            <View
              collapsable={undefined}
              style={
                Object {
                  "backgroundColor": "white",
                  "height": 20,
                  "width": 40,
                }
              }
            />
          </View>
          <View
            style={undefined}
          >
            <Image
              source={
                Object {
                  "testUri": "../../react-native-elements/src/rating/images/star.png",
                }
              }
              style={
                Object {
                  "height": 20,
                  "width": 20,
                }
              }
            />
          </View>
          <View
            style={undefined}
          >
            <Image
              source={
                Object {
                  "testUri": "../../react-native-elements/src/rating/images/star.png",
                }
              }
              style={
                Object {
                  "height": 20,
                  "width": 20,
                }
              }
            />
          </View>
          <View
            style={undefined}
          >
            <Image
              source={
                Object {
                  "testUri": "../../react-native-elements/src/rating/images/star.png",
                }
              }
              style={
                Object {
                  "height": 20,
                  "width": 20,
                }
              }
            />
          </View>
          <View
            style={undefined}
          >
            <Image
              source={
                Object {
                  "testUri": "../../react-native-elements/src/rating/images/star.png",
                }
              }
              style={
                Object {
                  "height": 20,
                  "width": 20,
                }
              }
            />
          </View>
          <View
            style={undefined}
          >
            <Image
              source={
                Object {
                  "testUri": "../../react-native-elements/src/rating/images/star.png",
                }
              }
              style={
                Object {
                  "height": 20,
                  "width": 20,
                }
              }
            />
          </View>
        </View>
      </View>
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Super great bar, would come back due to noise level
      </Text>
    </View>
  </View>
</View>
`;
